jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -0,0 +1,35 @@
+/******************************************************************************* 
+ * Copyright (c) 2022 Giancarlo Pernudi Segura and others. 
+ * 
+ * This program and the accompanying materials are made available under the 
+ * terms of the Eclipse Public License v. 2.0 which is available at 
+ * http://www.eclipse.org/legal/epl-2.0. 
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ * 
+ * Contributors: 
+ *     Giancarlo Pernudi Segura - initial API and implementation 
+ *******************************************************************************/
+
+package org.eclipse.lsp4jakarta.jdt.core.websocket;
+
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4jakarta.jdt.core.DiagnosticsCollector;
+import org.eclipse.lsp4jakarta.jdt.core.websocket.WebSocketConstants;
+import org.eclipse.jdt.core.ICompilationUnit;
+
+import java.util.List;
+
+public class WebSocketDiagnosticsCollector implements DiagnosticsCollector {
+    public WebSocketDiagnosticsCollector() {
+    }
+
+    public void completeDiagnostic(Diagnostic diagnostic) {
```

Comment:
Since it's implementing a method from the interface, we should add `@Override`

Reply:
I didn't put the `@Override` annotation since my understanding is I have to implement the abstract method so it's not the same as actually overriding a concrete method. I also based this on ServletDiagnosticsCollector which didn't use the `Override` annotation.
https://github.com/eclipse/lsp4jakarta/blob/2a7501e65ff171f76da30bd910b6107edf1ecbaa/jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/servlet/ServletDiagnosticsCollector.java#L49-L53
I can change it if @kathrynkodama agrees that it should have the  `@Override` annotation. What do you think?

Reply:
Even though you do not need the `@Override` annotation, I think it is best practice to include it. If anything for readability and to ensure that a compile time error is thrown if any of the method signatures are incorrect. 
Good catch on the ServletDiagnosticsCollector class - we should add `@Override` annotations there. Are you able to add that fix to this pull request?

Reply:
Yeah, I can add that in this PR it's no problem.